src/plasmapy/formulary/braginskii.py

```diff
@@ -323,8 +323,8 @@ def __init__(  # noqa: PLR0912, PLR0915
         m_i: u.Quantity[u.kg] = None,
         Z=None,
         B: u.Quantity[u.T] = 0.0 * u.T,
-        model="Braginskii",
-        field_orientation="parallel",
+        model: str = "Braginskii",
+        field_orientation: str = "parallel",
```

Comment:
📝 We might want to use `typing.Literal` here too, but I didn't for the time being since we're likely to revamp this file anyway...one of these years!